spec.md

```diff
@@ -841,11 +841,61 @@ For success responses, the following fields are defined:
 | --- | --- | --- |
 | dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. If present, MUST be a non-empty string. |
 | operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Last Operation](#polling-last-operation) endpoint in a percent-encoded query parameter. If present, MUST be a non-empty string. |
+| extension_apis | array-of-objects | For extensions to the Service Broker API, Service Brokers MAY return one or more `extension_api` objects that describe additional API endpoints via an OpenAPI document. See [Extension API Object](#extension-api-object) for more information. |
+
+##### Extension API Object
+
+The `extension_api` object MAY be used to describe any additional endpoint
+to the Open Service Broker API. An example of this could be lifecycle
+management of a Service Instance, (e.g. "Day Two Operations"), like Backup,
+Restore, Stop, Start, Restart and Pause.
+
+The `extension_api` MUST include a URI to an OpenAPI 3.0+ document that the
+Platform can use to determine the new endpoint(s), parameter(s), authentication
+mechanism and server URL. The new APIs are extensions to the Open Service Broker
+API. As such they are indended to be invoked by the Platform on behalf of its
+clients.
+
+Extension API endpoints MAY be executed on a remote server, however the OpenAPI
+document MUST include a Server Object with an accurate `url` parameter and a
+`description` field labeled, "Service Broker Extensions Server". If no Server
+Object `description` field contains, "Service Broker Extensions Server", the 
+Platform MUST assume the extension API endpoints are to be invoked using the
+Service Broker host and port. The `url` for this Server Object MUST be an
+absolute URL.
+
```

Comment:
I don't get this. If the description field is not "Service Broker Extensions Server", the platform uses the brokers host and port. Why need the URL be absolute in this case? Isn't it relative to the broker? What if the broker host/port don't match the URL?
It can be difficult for a broker to determine the right host name in this context. It would be easier for a broker to return the (absolute) path here.

Reply:
You're 100% right. I think I addressed this in the latest change.